include/zephyr/arch/arc/v2/linker.ld

```diff
@@ -93,6 +93,11 @@ SECTIONS {
 
 #include <zephyr/linker/common-rom.ld>
 #include <zephyr/linker/thread-local-storage.ld>
+#ifdef __MWDT_LINKER_CMD__
```

Comment:
Is this change related to the !XIP issue, or a more general fix/clean-up?
There may me other optional, included snippets that are not at the correct place currently. Maybe handle in a separate issue/PR? Or, try and ensure all issues are fixed in this PR, preferably in separate commits.

Reply:
This change related to !XIP issue. Here I just place RODATA_SECTION at the end of ROMABLE region. The gist is in the following. RODATA_SECTION defined explicitly in the script and it's quite convenient to set the section  border (and whole ROMABLE region border) with MPU_ALIGNMENT macro.